testnets/qwoyntestnet/chain.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "qwoyntestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://qwoyn.studio/",
+  "pretty_name": "Qwoyn",
+  "chain_id": "higgs-boson-1",
+  "bech32_prefix": "qwoyn",
+  "daemon_name": "qwoynd",
+  "node_home": "$HOME/.qwoynd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 123,
```

Comment:
Slip44 123 is not registered to qwoyn. Please correct this. Was 118, not correct?

Reply:
That's right, 118 was not correct.

Reply:
I'm confused though, I don't see any cosmos chains registered in slip44. I only see them in slip 173 and none of them are numbered.

Reply:
For SLIP44 (for address generation), the majority use the same method as Cosmos (118) and Ethereum (60)